ci/authn-azure/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+AWS_PEM_FILE_LOCATION="<location of pem file to login to machine>"
+USERNAME="<username of AWS machine>"
```

Comment:
I think better naming would be `AWS_USERNAME`

Reply:
or `AWS_INSTANCE_USERNAME`

Reply:
correct